scripts/build_and_restart_dgxie.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+set -xe
+
+
+compose_file=config/containers/dgxie/docker-compose-dgxie.yml
+compose_directory="."
+compose_cmd="docker-compose --project-directory ${compose_directory} -f ${compose_file}"
```

Comment:
I still think this adds unnecessary complexity, can we just leave the file called `docker-compose.yml` and avoid the script?